### PR TITLE
Web3 Websocket support

### PR DESCRIPTION
**Pull request by freddyli7**

### CODE REVIEW

File: action/evm.go [hunk 1]
```diff
@@ -78,8 +78,6 @@ func EthereumConfig(chainID string) *ethparams.ChainConfig {
 		MuirGlacierBlock:    big.NewInt(0),
 		BerlinBlock:         big.NewInt(0),
 		LondonBlock:         big.NewInt(0),
-
-		EWASMBlock: nil,
```
Comment (DeV1doR) [inline action/evm.go:82]: why?

File: web3/types/utils.go [hunk 2]
```diff
@@ -303,3 +305,45 @@ func StateAndHeaderByNumberOrHash(tmClient rpcclient.Client, blockNrOrHash rpc.B
 	}
 	return 0, errors.New("invalid arguments; neither block nor hash specified")
 }
+
+// ResultData represents the data returned in an sdk.Result
```
Comment (DeV1doR) [inline web3/types/utils.go:309]: remove `sdk.Result`

File: web3/websockets/service.go [hunk 3]
```diff
@@ -0,0 +1,319 @@
...
+					return
+				}
+
+				logs := rpcfilters.FilterLogs(resultData.Logs, crit.FromBlock, crit.ToBlock, crit.Addresses, crit.Topics)
+
+				api.filtersMu.Lock()
+				if f, found := api.filters[sub.ID()]; found {
+					// write to ws conn
+					res := &SubscriptionNotification{
+						Jsonrpc: "2.0",
```
Comment (DeV1doR) [inline web3/websockets/service.go:234]: request ID parameter missing?

File: action/evm.go (hunk 1 above)
Comment (freddyli7) [reply action/evm.go:82]: the new go-ethereum doenst support this anymore, so I removed it

**Pull request closed without merging**